code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -518,9 +518,9 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 
 	if(head?.inv_hide_flags & HIDE_EXCESS_HAIR)
 		var/image/mask = image('icons/mob/human_face.dmi', null, "Jeager_Mask")
-		mask.render_target = "*[REF(src)]"
+		mask.render_target = "*jaegFACE[REF(src)]"
 		hair_final.overlays += mask
-		hair_final.filters += filter(arglist(alpha_mask_filter(0, 0, null, "*[REF(src)]")))
+		hair_final.filters += filter(arglist(alpha_mask_filter(0, 0, null, "*jaegFACE[REF(src)]")))
```

Comment:
Noticed there may have been some cross contamination of render sources due to ref codes